db/seeds.js

```diff
@@ -67,32 +67,44 @@ function seeds() {
       author_id: 527,
       title: "yard sale",
       description: "everybody get out any items you wish to sell",
-      date: "07/29/2018"
+      category: "Sell",
```

Comment:
good call on adding categories. we'll have to be sure to update the files associated with this model. https://github.com/schmitty890/homestead/issues/31 we currently arent taking into account for category and location. this will now be a future story to implement on the partials/client side js associated with this. nice add on.